services/outputhost/messagecache.go

```diff
@@ -1069,9 +1070,11 @@ func (msgCache *cgMsgCache) isStalled() bool {
 	var m3St m3HealthState
 	var smartRetryDisabled bool
 
-	if strings.Contains(msgCache.GetOwnerEmail(), SmartRetryDisableString) {
+	msgCache.cgCache.extMutex.RLock()
```

Comment:
is the lock necessary to protect access to `dlqMerging` ? If so, can you make it an atomic int instead ?